src/TMedia.h

```diff
@@ -61,9 +62,9 @@ class TMediaPlayer
     }
     ~TMediaPlayer() = default;
 
-    TMediaData getMediaData() const { return mMediaData; }
+    TMediaData mediaData() const { return mMediaData; }
```

Comment:
:thinking: Is it worthwhile/workable to return a `const &`here?

Reply:
It was worth thr review for efficiency sake, but I see places in the code where updates will be made on the other side of this and it could have a negative impact.